src/ui/Analytics/AnalyticsActionListMeta.ts

```diff
@@ -330,6 +346,24 @@ export var analyticsActionCauseList = {
     name: 'breadcrumbFacet',
     type: 'breadcrumb'
   },
+  /**
+   * The search-as-you-type event that gets logged when a breadcrumb facet is selected and the query is updated.
+   *
+   * `actionCause`: `'breadcrumbMLFacet'`
```

Comment:
Maybe we should rename the actionCause to breadcrumbDynamicFacet...

In fact, perhaps we should change all those `mlFacet` / `MLFacet` instances to `dynamicFacet`.